app.js

```diff
@@ -0,0 +1,24 @@
+const { getTopics } = require("./controllers/nc-news.controller");
+const express = require("express");
+const app = express();
+
+
+app.use(express.json());
```

Comment:
Don't need this `app.use(express.json())` yet because we haven't added a post request yet, so can remove this from the PR 🙂 
It's best practise to only include code in a PR that meets the needs of the ticket 🎫